test/functional/api/v2/job_invocations_controller_test.rb

```diff
@@ -64,26 +64,27 @@ class JobInvocationsControllerTest < ActionController::TestCase
           assert_response :success
         end
 
-        test 'should create with feature' do
-          feature = FactoryBot.create(:remote_execution_feature,
-                                      :job_template => @template)
-          attrs = {
-            feature: feature.label,
-            host_ids: ['testfqdn'],
-            inputs: { 'plan_id' => 3, 'organization_id' => 10 }
-          }
-          # To avoid creating a host that has to be found on the
-          # JobInvocationComposer I just mock it here
-          Api::V2::JobInvocationsController.any_instance
-                                           .expects(:resource_finder)
-                                           .returns('testfqdn')
-          JobInvocationComposer.expects(:for_feature).with(
-            attrs[:feature],
-            anything,
-            has_entries('plan_id' => 3, 'organization_id' => 10)
-          ).returns(OpenStruct.new(trigger!: true, job_invocation: @invocation))
-          post :create, params: { job_invocation: attrs }
-          assert_response :success
+        context 'with_feature' do
+          setup do
+            @feature = FactoryBot.create(:remote_execution_feature,
+                                         :job_template => @template)
+            @attrs = {
+              feature: @feature.label
+            }
+          end
+
+          test 'host ids as array' do
```

Comment:
Could you add a specific case for array of ids? I know Katello is using ids, not fqdns: I tested and it seems to work, but I would rather have this covered by tests.